### PR TITLE
fix: patch-package the JSII bug

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Taking the JSII fix into the project early https://github.com/aws/jsii/pull/3866

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -811,7 +811,7 @@
     chalk "^4.1.2"
     semver "^7.3.8"
 
-"@jsii/spec@1.71.0", "@jsii/spec@^1.71.0":
+"@jsii/spec@1.71.0", "@jsii/spec@^1.68.0":
```
Comment (ansgarm) [inline yarn.lock:814]: Why did this spec version change?

File: patches/jsii-pacmak+1.71.0.patch [hunk 2]
```diff
@@ -0,0 +1,4180 @@
+diff --git a/node_modules/jsii-pacmak/bin/jsii-pacmak.js b/node_modules/jsii-pacmak/bin/jsii-pacmak.js
+index 96120e9..cb54420 100644
```
Comment (DanielMSchmidt) [inline patches/jsii-pacmak+1.71.0.patch:2]: weird, the diff shouldn't be this big

File: yarn.lock [hunk 3]
```diff
@@ -6744,7 +6765,7 @@ jsii-docgen@^7.0.157:
     semver "^7.3.8"
     yargs "^16.2.0"
 
-jsii-pacmak@^1.71.0:
+jsii-pacmak@^1.68.0:
```
Comment (ansgarm) [inline yarn.lock:6768]: Is this intended?

File: patches/jsii-pacmak+1.71.0.patch (hunk 2 above)
Comment (ansgarm) [reply patches/jsii-pacmak+1.71.0.patch:2]: You might've had 1.68.0 installed at the time of running the patch-package script? 🤔

Comment (DanielMSchmidt) [reply patches/jsii-pacmak+1.71.0.patch]: Yeah, patch-package took the package-lock.json file instead of the yarn.lock

File: .github/workflows/integration.yml [hunk 4]
```diff
@@ -40,7 +40,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
       - name: install dependencies
-        run: yarn install --frozen-lockfile --prefer-offline
+        run: yarn install --frozen-lockfile --prefer-offline && yarn run postinstall
```
Comment (ansgarm) [inline .github/workflows/integration.yml:43]: Just out of curiosity: Why is this required here but not locally?

**Review by ansgarm — APPROVED** [review]

File: .github/workflows/integration.yml (hunk 4 above)
Comment (DanielMSchmidt) [reply .github/workflows/integration.yml:43]: No idea, it's still not applying the patch, I'll just disable the test now to be able to debug the next steps further
